src/commands.rs

```diff
@@ -50,11 +50,21 @@ pub const DEFAULT_PORT: u16 = 4226;
 const SERVER_STARTUP_TIMEOUT: Duration = Duration::from_millis(10000);
 
 /// Get the port on which the server should listen.
-fn get_port() -> u16 {
-    env::var("SCCACHE_SERVER_PORT")
+fn get_addr() -> crate::net::SocketAddr {
+    #[cfg(unix)]
+    if let Ok(addr) = env::var("SCCACHE_SERVER_UDS") {
+        if let Ok(uds) = crate::net::SocketAddr::parse_uds(&addr) {
+            return uds;
+        }
+    }
+    let port = env::var("SCCACHE_SERVER_PORT")
         .ok()
         .and_then(|s| s.parse().ok())
-        .unwrap_or(DEFAULT_PORT)
+        .unwrap_or(DEFAULT_PORT);
+    crate::net::SocketAddr::Net(std::net::SocketAddr::new(
+        "127.0.0.1".parse().unwrap(),
```

Comment:
Should we use the `127.0.0.1` address here & inside `src/server.rs:567`? Is it OK to duplicate the default address in a few places?

Maybe we should add a `const &str` for this or just use the result of `get_addr()` inside the `src/server.rs`, isn't it?